source/basic.tex

```diff
@@ -5304,15 +5299,19 @@
 as described above, satisfy the resulting constraints as imposed here. \end{note}
 
 \pnum
-Two actions are \defn{potentially concurrent} if
+Two actions \defn{conflict} if one of them modifies a memory
+location\iref{intro.memory}, the other one reads or modifies the same
+memory location, and at least one of them is not atomic.
+
+\pnum
+Two actions are \defn{potentially concurrent} if they are unsequenced, and
```

Comment:
[intro.execution]/8:

> If A is not sequenced before B and B is not sequenced before A, then A and B are unsequenced.

"happens before" and "unsequenced" are not mutually exclusive.

Reply:
I know. Can you be more specific about what problem you see with the proposed wording?

Reply:
If A inter-thread happens before B, then A and B are unsequenced, and therefore by this definition they are potentially concurrent. And since you removed "neither happens before the other" from the data race definition, it's now a race.

Reply:
Ah, I see your point. Fixed.